src/frontend/src/pages/FinancePage/ReimbursementRequestDetailPage/ReimbursementRequestDetailsView.tsx

```diff
@@ -315,13 +316,20 @@ const ReimbursementRequestDetailsView: React.FC<ReimbursementRequestDetailsViewP
     }
   ];
 
+  const statuses = reimbursementRequest.reimbursementStatuses.map((status) => status.type);
+  const recentStatus = statuses[-1];
   return (
     <PageLayout
       title={`${
         isReimbursementRequestDenied(reimbursementRequest)
           ? `${fullNamePipe(reimbursementRequest.recipient)}'s Reimbursement Request - Denied`
           : `${fullNamePipe(reimbursementRequest.recipient)}'s Reimbursement Request`
       }`}
+      chips={
+        <Box id="status" display="flex" gap="20px">
```

Comment:
feel free to correct me if im missing something but i dont think you need this box

Reply:
I think the box is necessary. It throws a parsing error otherwise.